_includes/homepage-header.html

```diff
@@ -1,21 +1,15 @@
 <div class="row">
-
   <div class="col-sm-12">
-    <h1>Bringing <strong>mobile</strong> and <strong>enterprise</strong> together.</h1>
+    <h1>Cloud Native Mobile App Development on OpenShift.</h1>
   </div>  
+      
   <div class="col-sm-10 col-sm-offset-1">
-    <p><strong>We make cross-platform enterprise mobile development easy</strong>. Aerogear provides flexible, extensible libraries and server side components that will simplify your mobile development and infrastructure setup across platforms.</p>
+    <p><strong>We make cross-platform enterprise mobile development easy</strong>. AeroGear provides a suite of Mobile Services across a range of native and hybrid client SDKs that will simplify your mobile and modern application development needs, leveraging OpenShift's Container technology to provide a secure, scalable backend platform.</p>
```

Comment:
Not sure about having the `<strong>` element here. To me, if this is meant to be highlighted, it kind of just disappears into the paragraph. Maybe we can add it as a sub-title instead? Kind of something like this? wdyt?
![image](https://user-images.githubusercontent.com/9078522/41364796-9c643744-6f2f-11e8-91f3-a84902e0b28c.png)

Reply:
Yea agreed, it adds more emphasis